examples/crud-rest/package.json

```
@@ -0,0 +1,74 @@
+{
+  "name": "@loopback/example-crud-rest",
```

```suggestion
  "name": "@loopback/example-rest-crud",
```

The implementation package is named `@loopback/rest-crud`, your example `@loopback/example-crud-rest`.
Have you considered to use the same naming scheme?

---

I was considering that name too, but went with crud-rest because the component is `CrudRestComponent`, but I think it would be fine to change it to rest-crud to be consistent with the package name 👍